R/create.R

```diff
@@ -516,6 +516,24 @@ create_stan_data <- function(reported_cases, seeding_time,
   return(data)
 }
 
+##' Create initial conditions for delays
+##'
+##' @inheritParams create_initial_conditions
+##' @return A list of initial conditions for delays
+##' @keywords internal
+create_delay_inits <- function(data) {
```

Comment:
Something I have been playing around with is dropping this approach and instead sampling from the prior in the stan model. It would be nice as it would reduce duplication but it might be a bit unstable and also requires more stan code to be carried around for other tasks.